src/app/components/products/products-view.jsx

```diff
@@ -42,9 +38,18 @@ const useStyles = makeStyles({
     gridTemplateColumns: 'auto auto',
     alignItems: 'center',
   },
+  searchContainer: {
+    color: 'white!important',
+  },
   wrap: {
     flexFlow: 'wrap',
   },
+  '@global': {
+    '.MuiPaper-root.MuiPopover-paper.MuiPaper-elevation8.MuiPaper-rounded': {
+      width: 'min-content!important',
+      right: '600px!important',
```

Comment:
```suggestion
      width: 'min-content !important',
      right: '600px !important',
```

I've never seen the `!important` tag without a space before it